service/incremental-jsonsystem.py

```diff
@@ -73,6 +73,11 @@ def make_session(self):
             session.headers = self._config['headers']
         return session
 
+# to remove favicon not found errormessages in the log
+@app.route('/favicon.ico')
```

Comment:
Do we need it?

Reply:
we do not "need" it, but it removes annoying error messages from the logs